typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts

```diff
@@ -103,6 +103,11 @@ export interface Grid3DLayerProps extends ExtendedLayerProps {
      */
     colorMapFunction?: colorMapFunctionType | false;
 
+    /** Enable lines around cell faces.
+     *  default: true.
+     */
+    gridLines: boolean;
```

Comment:
Suggest making `gridLines` optional with the `?` operator

Reply:
The same prop isn't optional in triangle layer. For consistency, should all the props of the kind be optional?

Reply:
I think so, yes

Reply:
As it touches several layers suggest to make it in a separate task, if you don't mind.